invenio_communities/assets/semantic-ui/js/invenio_communities/search/components.js

```diff
@@ -0,0 +1,409 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+// Copyright (C) 2021-2022 New York University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { useState } from "react";
+import {
+  Button,
+  Card,
+  Checkbox,
+  Grid,
+  Icon,
+  Input,
+  Item,
+  Label,
+  List,
+  Message,
+  Segment,
+  Header,
+} from "semantic-ui-react";
+import { BucketAggregation, Toggle } from "react-searchkit";
+import _find from "lodash/find";
+import _get from "lodash/get";
+import _truncate from "lodash/truncate";
+import Overridable from "react-overridable";
+import { SearchBar } from "@js/invenio_search_ui/components";
+
+export const RDMRecordResultsListItem = ({ result, index }) => {
```

Comment:
@Pineirin I guess you left the same names as in invenio-app-rdm after discussion? I would say, just not to forget renaming them afterwards, to change the naming from `RDMRecord...` to `CommunityRecord...` in this file!